examples/pi/App.config

```diff
@@ -34,6 +34,19 @@
         <conversionPattern value="[%date] [%thread] [%-5level] [%logger] - %message%newline" />
       </layout>
     </appender>
-
   </log4net>
+
+  <appSettings>
+    <!--********************************************************************************************************-->
+    <!--** Uncomment the following settings to run Spark driver executable in **local** or **debug** modes ** -->
+    <!--** In debug mode, the driver is not launched by CSharpRunner but launched from VS or command prompt not configured for SparkCLR ** -->
+    <!--** CSharpBackend should be launched in debug mode as well and the port number from that should be used below ** -->
+    <!--** Command to launch CSharpBackend in debug mode is "sparkclr-submit.cmd debug" ** -->
+    <!--********************************************************************************************************-->
+
+    <!--
+    <add key="CSharpWorkerPath" value="C:\Git\SparkCLR\examples\Pi\bin\Debug\CSharpWorker.exe"/>
```

Comment:
In the case of example, exe is under "..\packages\Microsoft.SparkCLR.1.5.20\lib\net45\CSharpWorker.exe". It will be ideal to change the sample line pointing to this one under "..\packags".